src/libproc_macro/lib.rs

```diff
@@ -352,14 +352,14 @@ impl Span {
 
     /// Creates a new span with the same line/column information as `self` but
     /// that resolves symbols as though it were at `other`.
-    #[unstable(feature = "proc_macro_span", issue = "54725")]
+    #[stable(feature = "proc_macro_span_resolved_at", since = "1.43.0")]
```

Comment:
So maybe I'm being stupid but since this was merged on the 25th April shouldn't this be `since = "1.44.0"` what with 1.43.0 being released on the 23rd April

Reply:
With 1.43 released before this was merged, it's actually going to be stabilized in 1.45 (the GitHub milestone this PR is assigned to is also the one for 1.45). I've already opened #71574 to fix the attribute.

Reply:
So I'm not crazy? Good to know :-)